packages/icon/components/Icon.tsx

```diff
@@ -35,7 +36,7 @@ export interface IconProps {
 
 const Icon = (props: IconProps) => {
   const { color, size, shape, ariaLabel } = props;
-  const svgColor = color || textColorPrimary;
+  const svgColor = color || "currentColor";
```

Comment:
Should `"currentColor"` be a variable? Or how does this work?

Reply:
@natmegs - "currentColor" is a keyword in CSS that ends up being whatever color it would be if it was just a piece of text.
So if we had
```
.blueText {
    color: blue;
}
.blueText div {
    border-color: currentColor;
}
```
the border of any divs that are children of `.blueText` would be blue